docs/admin/kubeadm-alpha.md

```diff
@@ -0,0 +1,266 @@
+
+{% capture kubeconfig_all %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_kubeconfig_all.md %}
+{% endcapture %}
+
+{% capture kubeconfig_admin %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_kubeconfig_admin.md %}
+{% endcapture %}
+
+{% capture kubeconfig_kubelet %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_kubeconfig_kubelet.md %}
+{% endcapture %}
+
+{% capture kubeconfig_controller-manager %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_kubeconfig_controller-manager.md %}
+{% endcapture %}
+
+{% capture kubeconfig_scheduler %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_kubeconfig_scheduler.md %}
+{% endcapture %}
+
+{% capture kubeconfig_user %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_kubeconfig_user.md %}
+{% endcapture %}
+
+{% assign tab_set_name = "tab-kubeconfig" %}
+{% assign tab_names = "all,admin,kubelet,controller-manager,scheduler,user" | split: ',' | compact %}
+{% assign tab_contents = site.emptyArray | push: kubeconfig_all | push: kubeconfig_admin | push: kubeconfig_kubelet | push: kubeconfig_controller-manager | push: kubeconfig_scheduler | push: kubeconfig_user %}
+
+{% include tabs.md %}
+
+## Kubeadm alpha phase controlplane {#cmd-phase-controlplane}
+
+You can create all required static pod files for control plane components with `all` sub command, or 
+create selectively static pod file.
+
+{% capture controlplane_all %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_controlplane_all.md %}
+{% endcapture %}
+
+{% capture controlplane_apiserver %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_controlplane_apiserver.md %}
+{% endcapture %}
+
+{% capture controlplane_controller-manager %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_controlplane_controller-manager.md %}
+{% endcapture %}
+
+{% capture controlplane_scheduler %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_controlplane_scheduler.md %}
+{% endcapture %}
+
+{% assign tab_set_name = "tab-controlplane" %}
+{% assign tab_names = "all,apiserver,controller-manager,scheduler" | split: ',' | compact %}
+{% assign tab_contents = site.emptyArray | push: controlplane_all | push: controlplane_apiserver | push: controlplane_controller-manager | push: controlplane_scheduler %}
+
+{% include tabs.md %}
+
+## Kubeadm alpha phase etcd {#cmd-phase-etcd}
+
+Use following command to create a self-hosted, local etcd instance based on a static pod file.
+
+{% capture etcd-local %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_etcd_local.md %}
+{% endcapture %}
+
+{% assign tab_set_name = "tab-etcd" %}
+{% assign tab_names = "etcd local" | split: ',' | compact %}
+{% assign tab_contents = site.emptyArray | push: etcd-local %}
+
+{% include tabs.md %}
+
+## Kubeadm alpha phase mark-master {#cmd-phase-mark-master}
+
+Use following command to add master Label and Taint to the node.
```

Comment:
Use the following command to label the node with the `node-role.kubernetes.io/master` taint